### ISSUE TITLE
Startup: MQTT "The profile for the user is a temporary profile" exceptions cause GM to never reach the Start Page

### ISSUE THREAD

**Issue by iampremo** [YoYoGames/GameMaker-Bugs#948]
Repeated "MQTT - System.AggregateException: One or more errors occurred. (The profile for the user is a temporary profile.)" errors can cause GM to never reach the Start Page, causing a "blank" GM window to be shown until the user closes the IDE themselves:

![image](https://github.com/YoYoGames/GameMaker-Bugs/assets/15669932/05694669-3cd1-49ad-b041-2a1e152e9f24)

The ui.log is absolutely filled with errors like this one:

```
{quote}[17:33:57:616(5849)] MQTT - System.AggregateException: One or more errors occurred. (The profile for the user is a temporary profile.)
---> Internal.Cryptography.CryptoThrowHelper+WindowsCryptographicException: The profile for the user is a temporary profile.
at Internal.Cryptography.Pal.CertificatePal.FilterPFXStore(ReadOnlySpan1 rawData, SafePasswordHandle password, PfxCertStoreFlags pfxCertStoreFlags) at Internal.Cryptography.Pal.CertificatePal.FromBlobOrFile(ReadOnlySpan1 rawData, String fileName, SafePasswordHandle password, X509KeyStorageFlags keyStorageFlags)
at System.Security.Cryptography.X509Certificates.X509Certificate..ctor(ReadOnlySpan`1 data)
at System.Security.Cryptography.X509Certificates.X509Certificate2..ctor(Byte[] rawData)
at MQTTnet.Certificates.BlobCertificateProvider.GetCertificate()
at MQTTnet.Implementations.MqttTcpServerAdapter.StartAsync(IMqttServerOptions options)
at MQTTnet.Server.MqttServer.StartAsync(IMqttServerOptions options)
--- End of inner exception stack trace ---
at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
at YoYoStudio.IDE.MqttInitialise(){quote}
```
[Win11MQTTCrash.log](https://github.com/YoYoGames/GameMaker-Bugs/files/12726666/Win11MQTTCrash.log)

_The customer later advised they were able to fix their issue by running Windows Update and installing update KB5022360 for Windows 11._

### Steps (not reproduced yet):
1. Set up a PC with a fresh Win 11 installation and do not run Windows Update yet.
2. Install and try to start GM.
3. Observe if you get the issue.

**Comment by YYDan** [comment]
tickets/212516 is this also, but on Windows 10

**Comment by rwkay** [comment]
a fix has been added in same commit for #1784

**Comment by YYDan** [comment]
That fix is now in 2023.8.2 and is pretty small and looks good - however, we had never reproduced the original issue, so we will leave the report open a little longer and see if any of the people affected by this issue confirm it can be closed / make sure we do not get new reports using 2023.8.2.
